configuration/messageproperties/messages_en.properties

```diff
@@ -1529,13 +1529,20 @@ pihcore.mealsPerDay=Meals per day
 pihcore.numberPerDay=Number per day
 zl.socioEconomic.monthsLivingInHaiti=Number of months residing in Haiti
 
-
-pihcore.mentalhealth.generalAppearance =General appearance wnl
-pihcore.mentalhealth.speech=Speech wnl
-pihcore.mentalhealth.cognitive=Cognitive function wnl
+pihcore.mentalhealth.status=Status
```

Comment:
You do not need this.  Use "pihcore.status"

Reply:
@lnball I made the changes asked

Reply:
If you aren't using this now, can you remove this line?

Reply:
@lnball I remove pihcore.mentalhealth.status